### DISCUSSION TITLE
v0.3.0 bug Action Mannager cannot use

### DISCUSSION THREAD

**Discussion by ntdsir** [nextai-translator/nextai-translator#1172] [General]
v0.3.0 bug Action Mannager cannot use
![bug](https://github.com/openai-translator/openai-translator/assets/128882888/6a5960e0-da9a-4c84-8f09-110b94799b57)

**Comment by Josh-Archer** [comment]
### 📢 Roadmap Update & Consolidated Status

We have consolidated the 14-phase roadmap into a clean, concise progression tracking our progress to `v1.0.0`:

| Phase | Target Version | Focus Area | Status |
|:---|:---|:---|:---|
| **Phase 1: Quality Foundation** | `v0.31.0` | Core schema verification, CI fast merge gates, test harness. | ✅ Completed |
| **Phase 2: Observability** | `v0.32.0` | Diagnostic data sources, service status, media inspection. | ✅ Completed |
| **Phase 3: IAM & Access** | `v0.33.0` | User lifecycle, bitmask permissions (`seerr_permission_set`), watchlists. | ✅ Completed |
| **Phase 4: Reference Helpers** | `v0.34.0` | TMDB genres, languages, and regional lookup data sources. | ✅ Completed |
| **Phase 5: Provider Hardening** | `v0.35.0` | `ImportState` implementations, input validators, plan modifiers. | ✅ Completed |
| **Phase 6: Advanced Lifecycle** | `v0.36.0` | Request approvals/declines, issue comments, user metadata. | ✅ Completed |
| **Phase 7: Servarr Resolvers** | `v0.37.1` | Dynamic Radarr/Sonarr entity resolution (profiles, root folders, tags). | ✅ Completed |
| **Phase 8: Dual Registry & Automation** | `v0.38.2` | Terraform + OpenTofu dual-publishing, Release Please, daily OpenAPI drift watcher. | ✅ Completed |
| **Phase 9: Community Readiness** | `v0.39.0` | Contributor onboarding (`CONTRIBUTING.md`), issue/PR templates, devcontainer ([#183](https://github.com/Josh-Archer/terraform-provider-seerr/issues/183)). | 🔵 **In Progress** |
| **Phase 10: Import & Migration Tooling** | `v0.40.0` | Bulk CLI generator (`tools/importer`) generating HCL & import blocks from live instances ([#170](https://github.com/Josh-Archer/terraform-provider-seerr/issues/170)). | 📅 Planned |
| **Phase 11: State Resilience & Upgraders** | `v0.41.0` | Schema upgraders, semantic diff suppression, uniform 404 state removal ([#169](https://github.com/Josh-Archer/terraform-provider-seerr/issues/169)). | 📅 Planned |
| **Phase 12: Observability & Disaster Recovery** | `v0.42.0` | Backup/recovery runbooks, drift detection dashboards, metrics presets ([#184](https://github.com/Josh-Archer/terraform-provider-seerr/issues/184)). | 📅 Planned |
| **Phase 13: Module Ecosystem** | `v0.43.0` | Turnkey composite modules for media server stacks and arr integrations ([#171](https://github.com/Josh-Archer/terraform-provider-seerr/issues/171)). | 📅 Planned |
| **Phase 14: v1.0.0 General Availability** | `v1.0.0` | API stability guarantee, breaking change policy, cross-engine acceptance matrix. | 🏁 Planned |

```text
v0.38.3 (Current) ➔ v0.39.0 (Community) ➔ v0.40.0 (Bulk Import) ➔ v0.41.0 (Resilience) ➔ v0.42.0 (DR) ➔ v0.43.0 (Modules) ➔ v1.0.0 (GA)
```

See full details in [`ROADMAP.md`](https://github.com/Josh-Archer/terraform-provider-seerr/blob/main/ROADMAP.md).
